resources/decision_tree_GRCh38.json

```
@@ -397,6 +407,100 @@
       "outcomeFalse": {
         "nextNode": "exit_lb"
       },
+      "outcomeMissing": {
+        "nextNode": "non_coding"
+      }
+    },
+    "non_coding": {
+      "description": "Non coding tools above threshold",
```

```suggestion
      "description": "ReMM, ncER or FATHMM_MKL_NC above threshold",
```
since there is another non-coding tool now